Example-Crypto-REXX/README.md

```diff
@@ -0,0 +1,107 @@
+## Example-Crypto-REXX
+
+This sample queries Image Activation profiles to list crypto attributes for LPARs. It stores the results, which are in .csv format, in a z/OS data set member.  You can import the content into Microsoft™ Excel or another application of your choice.
+
+This sample uses **HWIREST API** to do the following:
+- List CPCs and retrieve the URI and target name associated with the LOCAL CPC or the *CPCname* provided
+- List the LPARs on the designated CPC and query the LPAR status.
+- For each LPAR matching the requested status (default is operating), retrieve the following information from the image activation profile:
+(assumption is the image activation profile name matches the LPAR name)
```

Comment:
For each LPAR matching the requested status (default is operating), access the image activation profile with the same name as the LPAR to retrieve the following information:

Reply:
Done.